base_technical_features/models/res_users.py

```diff
@@ -51,5 +51,5 @@ def _inverse_technical_features(self):
 
     def __init__(self, pool, cr):
         super().__init__(pool, cr)
-        self.SELF_READABLE_FIELDS += ["technical_features", "show_technical_features"]
-        self.SELF_WRITEABLE_FIELDS.append("technical_features")
+        type(self).SELF_READABLE_FIELDS += ["technical_features", "show_technical_features"]
+        type(self).SELF_WRITEABLE_FIELDS.append("technical_features")
```

Comment:
maybe using properties could be clearer?

```
@property
def SELF_READABLE_FIELDS(self):
    return super().SELF_READABLE_FIELDS + ["technical_features", "show_technical_features"]

@property
def SELF_WRITEABLE_FIELDS(self):
    return super().SELF_WRITEABLE_FIELDS + ["technical_features"]

```

Reply:
That would indeed be clearer that it overrides a property.
I'm just wondering, as this notation is used everywhere when overriding these exact same properties, would it be wise to use a different notation ?

Reply:
> as this notation is used everywhere when overriding these exact same properties,

Oh, then it makes sense to use the same as the other places.
Thanks for the clarification!